### PR TITLE
doc: Added documentation folder for readability and maintenance

### PR DESCRIPTION
**Pull request by leemgs**
This commit is to append the documentation folder newly to maintain the README.md 
file with high-readability. It is because the file is a gateway of the FauxPilot project, let's 
simplify the contents of the README.md file ASAP.

Signed-off-by: Geunsik Lim <leemgs@gmail.com>
Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

### CODE REVIEW

**Comment by leemgs** [comment]
### After this PR

image

File: README.md [hunk 1]
```diff
@@ -14,206 +15,39 @@ You'll need:
...
-
-Another issue with using the Copilot plugin is that its tokenizer (the component that turns text into a sequence of integers for the model) is slightly different from the one used by CodeGen, so the plugin will sometimes send a request that is longer than CodeGen can handle. You can work around this by replacing the `vocab.bpe` and `tokenizer.json` found in the Copilot extension (something like `.vscode/extensions/github.copilot-[version]/dist/`) with the ones found [here](https://github.com/moyix/fauxpilot/tree/main/copilot_proxy/cgtok/openai_format).
-
-Have fun!
+Please refer to [How to set-up a FauxPilot server](documentation/server.md).
+
+
+### Client configuration for FauxPilot
+
+We offer some ways to connect to FAUXPILOT Server. For example, you can create a client by how to open the Openai API, Copilot Plugin, REST API.
```
Comment (moyix) [inline README.md:40]: This should be FauxPilot I think

File: documentation/client.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+We offer some ways to connect to FAUXPILOT Server. For example, you can create a client by how to open the Openai API, Copilot Plugin, REST API.
```
Comment (moyix) [inline documentation/client.md:1]: Ditto here, s/FAUXPILOT/FauxPilot/

Comment (leemgs) [reply documentation/client.md:1]: > Ditto here, s/FAUXPILOT/FauxPilot/

Fixed. Thanks.

**Comment by moyix** [comment]
Great, thanks for the documentation PR! :)

**Pull request merged**